.travis.yml

```diff
@@ -29,7 +29,6 @@ install:
   - pip install nose_parameterized
   #- pip install --no-deps git+https://github.com/quantopian/zipline
   - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then conda install --yes mock enum34; fi
-  - pip install --no-deps git+https://github.com/pymc-devs/pymc3.git
```

Comment:
pymc3 is installed by `pip install .[bayesian]`. There's no need to install it twice. This dev version was also missing `pymc3.sqrt` and causing some tests to fail.